components/scream/src/physics/shoc/tests/shoc_tke_isotropic_ts_tests.cpp

```diff
@@ -175,8 +175,55 @@ struct UnitWrap::UnitTest<D>::TestShocIsotropicTs {
 
   static void run_bfb()
   {
-    // TODO
-  }
+    SHOCIsotropicData f90_data[] = {
```

Comment:
This struct has been regenerated with a new name, so so minor changes will be needed here.